### PR TITLE
Fixed issue with using AbstractEnvironment within CartPole etc

### PR DESCRIPTION
**Pull request by darsnack**

### CODE REVIEW

File: src/envs/cartpole.jl [hunk 1]
```diff
@@ -1,9 +1,28 @@
...
 #		Copied from https://webdocs.cs.ualberta.ca/~sutton/book/code/pole.c
 
+using Reinforce: AbstractEnvironment
+using LearnBase: DiscreteSet
+using RecipesBase
+@static if VERSION >= v"0.7"
+  using Random: seed!
+else
+  seed! = srand
+end
```
Comment (iblislin) [inline src/envs/cartpole.jl]: I think we already dropped the v0.6 support. So just `using Random: seed!`

Comment (darsnack) [reply src/envs/cartpole.jl]: Pushed a new commit to reflect this.

File: src/envs/cartpole.jl [hunk 2]
```diff
@@ -1,9 +1,24 @@
...
+
+import Reinforce: reset!, actions, finished, step!
+
+export
+  CartPole,
+  reset!,
+  step!,
+  actions,
+  finished,
+  f
```
Comment (iblislin) [inline src/envs/cartpole.jl]: `f` ?
I think recipe can work without exporting it. Could you test that?

Comment (darsnack) [reply src/envs/cartpole.jl]: Yup, this works. Committed.

**Comment by iblislin** [comment]
:+1:

**Pull request merged**